product-service/models/initDB.sql

```diff
@@ -0,0 +1,35 @@
+create extension if not exists "uuid-ossp";
+
+create table if not exists products (
+	id uuid primary key default uuid_generate_v4(),
+	title text,
+	description text,
+	price integer,
+	imageUrl varchar(255)
+);
+
+create table if not exists stocks (
+	product_id uuid references products (id),
```

Comment:
You could set `product_id` as a primary key for this table

Reply:
thanks for your review, fixed